resources/puppetlabs/lein-ezbake/template/global/ext/redhat/preinst.erb

```diff
@@ -21,8 +21,6 @@ else
 <% end -%>
   useradd "${useradd_options[@]}" <%= EZBake::Config[:user] %> || :
 fi
-<% if defined?(additional_preinst)
-additional_preinst.split(',').each do |cmd| -%>
+<% EZBake::Config[:debian][:additional_preinst].each do |cmd| -%>
```

Comment:
What happens if there are no additional preinst tasks? Does this end up an empty array or nil?

Reply:
it will be an empty array https://github.com/puppetlabs/ezbake/blob/master/resources/puppetlabs/lein-ezbake/staging-templates/ezbake.rb.mustache#L55